profile/README.md

```diff
@@ -13,9 +13,9 @@ chains.
 - Check out the introduction to
   [building on ZetaChain](https://www.zetachain.com/docs/developers/overview/)
 - Follow tutorials to create your
-  [first omnichain contract](https://www.zetachain.com/docs/developers/omnichain/tutorials/hello/)
+  [first omnichain contract](UPDATE_LINK)
   or
-  [a cross-chain messaging contract](https://www.zetachain.com/docs/developers/cross-chain-messaging/examples/hello-world/)
+  [a cross-chain messaging contract](UPDATE_LINK)
```

Comment:
Feel free to remove cross-chain messaging as we no longer support connector-based messaging.